klient/k8s/resources/Cluster_with_param_test.go

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
It would be good if have this under examples folder. 

It is a good practice to keep the the actual code and only the UT for respective code together.